pkg/clusterapi/system.go

```diff
@@ -97,6 +122,16 @@ func (c *system) Run(ctx context.Context) error { //nolint:gocyclo
 
 	// Create the local control plane.
 	lcp := &localControlPlane{}
+
+	ipv4, err := hostHasIPv4Address()
+	if err != nil {
+		return err
+	}
+	// If the host has no IPv4 available, the default value of service network should be modified to IPv6 CIDR.
+	if !ipv4 {
+		lcp.APIServerArgs = []string{"--service-cluster-ip-range=fd02::/112"}
```

Comment:
I took a round IPv6 CIDR to add as value. Not sure if there is another one which is generally used as default